test/components/NavBar/OidcLoginComponent.test.jsx

```diff
@@ -4,6 +4,16 @@ import React from 'react';
 import { expect, it, vi, afterEach } from 'vitest';
 import OidcLoginComponent from '../../../src/components/NavBar/OidcLoginComponent';
 
+vi.mock('@inrupt/solid-ui-react', () => ({
```

Comment:
I learned that vi actually hoists these `vi.mock` calls to the top of the file, so putting them in the `describe` block doesn't do anything.